api/github.js

```
@@ -142,116 +157,108 @@ export default async function handler(req) {
   const cached = CACHE.get(repo);
-  if (cached && Date.now() - cached.ts < CACHE_TTL) {
-    return new Response(JSON.stringify({ ...cached, cached: true }), { status: 200, headers });
-  }
+  if (cached && Date.now() - cached.ts < CACHE_TTL) return new Response(JSON.stringify({ ...cached, cached: true }), { status: 200, headers });
 
   try {
     const [repoRes, commitsRes] = await Promise.all([
       fetchWithFallback(`https://api.github.com/repos/${repo}`, { headers: ghHeaders }),
       fetchWithFallback(`https://api.github.com/repos/${repo}/commits?per_page=1`, { headers: ghHeaders }),
     ]);
+    const rh = addRateLimitHeaders(repoRes, { ...headers });
```

ESLint

[error] 'addRateLimitHeaders' is not defined.

(no-undef)